src/data_api/orders.se

```diff
@@ -155,105 +155,79 @@ def getWorstBidOrderID(market: address, outcome: uint256):
 def getWorstAskOrderID(market: address, outcome: uint256):
     return(self.worstOrder[market][outcome].ask: address)
 
-# @private
-def updateBestBidOrder(orderID: address, market: address, fxpPrice, outcome: uint256, betterOrderID: uint256):
-    if(msg.sender != self):
-        ~invalid()
-    bestBidOrderID = self.bestOrder[market][outcome].bid
-    # If best bid is not set, this order is the new best bid
-    if(bestBidOrderID == 0):
-        self.bestOrder[market][outcome].bid = orderID
-    else:
-        # If price higher than best bid price, this order is the new best bid
-        if(fxpPrice > self.orders[bestBidOrderID].fxpPrice):
-            if(betterOrderID != 0):
-                ~invalid()
-            self.bestOrder[market][outcome].bid = orderID
-        # If price is lower than the best bid price:
-        #  - betterOrderID must be specified
-        #  - price must be less than or equal to betterOrderID's price
-        else:
-            if(betterOrderID == 0):
-                ~invalid()
-            if(fxpPrice > self.orders[betterOrderID].fxpPrice):
-                ~invalid()
-    return(1: uint256)
+# @public
+def isBetterPrice(type: uint256, fxpPrice, orderID: address):
```

Comment:
The convention I've been using is methods that are just getters (i.e., that don't modify any contract data) don't need a mutex and therefore also don't use the `public` prefix.